Gemfile

```diff
@@ -169,15 +169,14 @@ gem 'http_accept_language'
 # User authentication
 gem 'devise'
 gem 'devise_masquerade'
-# TODO: To remove restriction once v2.0 stabilises.
-gem 'devise-multi_email', '~>1.0.5'
+gem 'devise-multi_email'
 
 gem 'omniauth'
 gem 'omniauth-facebook'
 
 # Use cancancan for authorization
 gem 'cancancan'
-gem 'cancancan-squeel'
+gem 'cancancan-baby_squeel', github: 'jeremyyap/cancancan-baby_squeel'
```

Comment:
Can suggest to put a TODO to actually switch this to a gem under Coursemology repo instead, and explain the purpose of this gem (fix self-join issues and prevent eager loading).